modules/core/core-scheduler/src/main/java/com/enonic/xp/impl/scheduler/SchedulerConfigImpl.java

```diff
@@ -0,0 +1,187 @@
+package com.enonic.xp.impl.scheduler;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.TimeZone;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.enonic.xp.config.ConfigBuilder;
+import com.enonic.xp.config.ConfigInterpolator;
+import com.enonic.xp.config.Configuration;
+import com.enonic.xp.form.PropertyTreeMarshallerService;
+import com.enonic.xp.page.DescriptorKey;
+import com.enonic.xp.scheduler.CalendarService;
+import com.enonic.xp.scheduler.CreateScheduledJobParams;
+import com.enonic.xp.scheduler.ScheduleCalendar;
+import com.enonic.xp.scheduler.ScheduleCalendarType;
+import com.enonic.xp.scheduler.SchedulerName;
+import com.enonic.xp.security.PrincipalKey;
+
+@Component(configurationPid = "com.enonic.xp.scheduler")
```

Comment:
@vbradnitski missing the `com.enonic.xp.scheduler.cfg` file in the `https://github.com/enonic/xp/tree/epic-scheduler/modules/runtime/src/home/config` folder. Please add it.

Reply:
it's not missing, it's there

Reply:
Yes, I can see now. Please, do not forget to point which properties should be used for a job configuration.